composer.json

```diff
@@ -11,7 +11,7 @@
     "php" : ">=5.6.0"
   },
   "require-dev" : {
-    "xp-framework/unittest": "^9.0 | ^8.0 | ^7.0 | ^6.5"
+    "xp-framework/unittest": "dev-feature/baseless-single-instance as 10.0.0"
```

Comment:
This can be set to `^10.0` once the feature has been merged and unittesting library has been released